docs/en/02_Developer_Guides/00_Model/12_Indexes.md

```diff
@@ -27,8 +27,8 @@ support the following:
  * `unique`: Index plus uniqueness constraint on the value
  * `fulltext`: Fulltext content index
 
-In order to use more database specific or complex index notations, we also support raw SQL for as a value in the 
-`$indexes` definition. Keep in mind this will likely make your code less portable between databases.
+In order to use more database specific or complex index notations, we also support raw SQL as a value in the 
+`$indexes` definition. Keeping this in mind will likely make your code less portable between databases.
```

Comment:
I'm not sure this is the correct change of wording here.

> Keep in mind that using raw SQL is likely to make your code less portable between DBMSs

is probably about right